### DISCUSSION TITLE
Spin up a new database on-the-fly

### DISCUSSION THREAD

**Discussion by JoeCianflone** [planetscale/discussion#306] [Q&A] [answered]
Hi, I've got a multi-tenant, multi-database app and I'm wondering if this service would work for it. When a new user registers that kicks off a job that will end with it creating a new database and running migrations and seeding some data.

All new users would be under hobby for sure, so is there a way to do this? On top of that, since I have lots of small databases is there a way to pick some metric and programmaticly scale up the DB once it hits that? 

Basically, I'd like to not have to think about scaling each individual user if possible.

**Comment by orware** [comment, marked as answer]
Hi @JoeCianflone,

Generally this particular use case isn't recommended at this time since it'll likely result in excessive costs for you.

Under this setup only one of the databases would be allowed to stay on the Hobby plan, and the others would have to be created on the Scaler plan. 

Manageability could also be a factor here if we're talking about hundreds/thousands of users and individual databases being created within your PlanetScale organization.

The more ideal database scenario in this case is a multi-tenant setup that uses a single database that way the usage/resources can be more focused on that specific database within our service, and is also where our ability to scale is able to truly shine, but I understand if your existing application cannot be retrofitted to work under a single database easily.

I hope this information is helpful in providing some guidance!
